stable/atlantis/values.yaml

```
@@ -195,6 +195,8 @@ serviceAccount:
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
+  # Annotations for the Service Account
+  serviceAccountAnnotations: {}
```

```suggestion
  annotations: {}
```

Don't need serviceAccount prefix since this is already under the serviceAccount key